pkg/products/rhsso/reconciler.go

```diff
@@ -779,7 +779,7 @@ func OsUserInKc(osUsers []usersv1.User, kcUser keycloak.KeycloakAPIUser) bool {
 	return false
 }
 
-func (r *Reconciler) createOrUpdateKeycloakUser(user keycloak.KeycloakAPIUser, inst *integreatlyv1alpha1.Installation, ctx context.Context, serverClient k8sclient.Client) (controllerutil.OperationResult, error) {
+func (r *Reconciler) createOrUpdateKeycloakUser(ctx context.Context, inst *integreatlyv1alpha1.Installation, user keycloak.KeycloakAPIUser, serverClient k8sclient.Client) (controllerutil.OperationResult, error) {
```

Comment:
Similar to above, the `user` param doesn't need to be moved to after the `inst` one I think. So it could instead be:

```suggestion
func (r *Reconciler) createOrUpdateKeycloakUser(ctx context.Context, user keycloak.KeycloakAPIUser, inst *integreatlyv1alpha1.Installation, serverClient k8sclient.Client) (controllerutil.OperationResult, error) {
```